pkg/plugins/gateway/lambda/lambda.go

```diff
@@ -123,6 +142,30 @@ func (s *LambdaGateway) triggersFromRequest(data map[string]interface{}) ([]trig
 				}
 			}
 		}
+	case cloudwatch:
+		evt := &events.CloudWatchEvent{}
+		err := json.Unmarshal(bytes, evt)
+
+		if err != nil {
+			return nil, fmt.Errorf("unable to unmarshal cloudwatchEvent: %v", err)
+		}
+
+		evtRuleArn := evt.Resources[0]
+
+		sched, err := s.getScheduleNameForArn(evtRuleArn)
+
+		if err != nil {
+			log.Default().Println("unable to locate nitric schedule")
+			return nil, fmt.Errorf("unable to find nitric schedule: %v", err)
+		}
+
+		log.Default().Println("creating event from cloud watch")
+
+		trigs = append(trigs, &triggers.Event{
+			ID:      "TODO",
```

Comment:
isn't uuid.NewString() better than TODO?

Reply:
@asalkeld addressed this with generation based on timestamp (millis from schedule execution time.